### PR TITLE
feat(addon-logs-archives): add AddonLogsArchives method

### PR DESCRIPTION
**Pull request by sihamais**
- [x] Add a [changelog entry](https://changelog.scalingo.com/)

Related to Scalingo/cli#518

### CODE REVIEW

**Comment by sihamais** [comment]
Should I remove `LogsArchivesByCursor` while I'm at it ? @EtienneM

File: addons.go [hunk 1]
```diff
@@ -16,6 +18,7 @@ type AddonsService interface {
 	AddonUpgrade(app, addonID string, params AddonUpgradeParams) (AddonRes, error)
 	AddonToken(app, addonID string) (string, error)
 	AddonLogsURL(app, addonID string) (string, error)
+	AddonLogsArchives(app, addonId string, page int)(*LogsArchivesResponse, error)
```
Comment (EtienneM) [inline addons.go:21]: I think this method should belong to the `LogsArchivesService`, don't you think?

**Review by EtienneM — COMMENTED** [review]
Neither the PR title, nor the commits message respect the convention (https://karma-runner.github.io/6.3/dev/git-commit-msg.html).

File: addons.go (hunk 1 above)
Comment (EtienneM) [reply addons.go:21]: OK you convinced me 👍🏾

**Review by EtienneM — APPROVED** [review]
Did you already tested it on the CLI PR?

**Review by EtienneM — APPROVED** [review]

**Pull request merged**